x/config/config.go

```diff
@@ -93,6 +94,14 @@ func newStreamDialerFromPart(innerDialer transport.StreamDialer, oneDialerConfig
 	case "tls":
 		return newTlsStreamDialerFromURL(innerDialer, url)
 
+	case "tlsfrag":
+		fixedBytesStr := url.Opaque
+		fixedBytes, err := strconv.Atoi(fixedBytesStr)
+		if err != nil {
+			return nil, fmt.Errorf("invalid tlsfrag option: %v. It should be in tlsfrag:<number> format", fixedBytesStr)
+		}
+		return tlsfrag.NewFixedBytesStreamDialer(innerDialer, fixedBytes)
```

Comment:
This PR doesn't necessarily have to block on https://github.com/Jigsaw-Code/outline-sdk/pull/134, but I guess it's fine.